api/http/device.go

```diff
@@ -135,7 +136,7 @@ func (h DeviceController) Connect(c *gin.Context) {
 	}
```

Comment:
Can you change the `Upgrader.Subprotocols` from `[]string{"binary"}` to `[]string{"protomsg/msgpack"}`?

Reply:
[sure](https://github.com/mendersoftware/deviceconnect/pull/13/commits/03dd6c7c927d58219cb608e8f1f708c78bd11362) man